infrastructure/identity-provider/README.md

```diff
@@ -118,15 +118,16 @@ Here there is the commands for Linux (Bash/Zsh):
 
 ```
 (
-  set -x; cd "$(mktemp -d)" &&
-  curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.{tar.gz,yaml}" &&
-  tar zxvf krew.tar.gz &&
-  KREW=./krew-"$(uname | tr '[:upper:]' '[:lower:]')_amd64" &&
-  "$KREW" install --manifest=krew.yaml --archive=krew.tar.gz &&
-  "$KREW" update
+ set -x; cd "$(mktemp -d)" &&
+ OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+ ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+ KREW="krew-${OS}_${ARCH}" &&
+ curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/${KREW}.tar.gz" &&
+ tar zxvf "${KREW}.tar.gz" &&
+ ./"${KREW}" install krew
 )
  ```
-
+Replace amd64 with arm64 in the ARCH field if you have a Macbook with M1 processsor.
```

Comment:
Isn't this automatically done by the command above?

Reply:
Yeah, sorry